release/tablelist.js

```diff
@@ -4462,6 +4467,17 @@ angular
         return Venue.listInventory(options).$promise;
       };
 
+      VenueService.prototype.listActiveInventory = function(options) {
```

Comment:
personally I'm a fan of passing `venueId` as first argument and options as second. We've gone back and forth on this so I'll leave it up to your discretion